extensions/positron-python/.github/workflows/build.yml

```diff
@@ -355,7 +355,7 @@ jobs:
         env:
           TEST_FILES_SUFFIX: testvirtualenvs
           CI_PYTHON_VERSION: ${{ matrix.python }}
-        uses: GabrielBB/xvfb-action@v1.6
+        uses: GabrielBB/xvfb-action@v1.7
```

Comment:
We could bump this in our CI too

Reply:
It looks like we are already using `GabrielBB/xvfb-action@v1.7` in `positron-python-ci.yml` 🎉